### PR TITLE
Added recognition mode for code and text. Fixed properties doc.

### PR DESCRIPTION
**Pull request by moritzhartmeier**
Also removed camera2 disabling from properties (why was it even there?).

### CODE REVIEW

File: docs/Scandit.java [hunk 1]
```diff
@@ -1026,19 +1040,6 @@ public void show(function didScan, function didManualSearch=null, function didCa
          */
         public int maxNumberOfCodesPerFrame;
 
-        /**
-         * Sets a property.
-         * <p>
-         * This function is for internal use only and any functionality that can be
```
Comment (robertnawrot) [inline docs/Scandit.java]: Is it still a function tho? 😄

File: src/settings.js [hunk 2]
```diff
@@ -111,7 +109,8 @@ function ScanSettings() {
 
 ScanSettings.RecognitionMode = {
 	TEXT: 1,
-	CODE: 2
+    CODE: 2,
```
Comment (robertnawrot) [inline src/settings.js]: Is it just github or is this indentation somehow different?

File: docs/Scandit.java (hunk 1 above)
Comment (moritzhartmeier) [reply docs/Scandit.java]: definitely not :D

Comment (moritzhartmeier) [reply docs/Scandit.java]: fixed

File: src/settings.js (hunk 2 above)
Comment (moritzhartmeier) [reply src/settings.js]: there were tabs there before, replaced all by spaces

**Pull request merged**